javascript/new-src/src/app.js

```diff
@@ -1,4 +1,4 @@
-/**
+/*!
```

Comment:
No, it's to prevent the header from being dropped by the concat job that creates app.js